src/test/resources/refaster/ArraysRecipe.java

```diff
@@ -53,8 +60,12 @@ public String getDescription() {
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
         JavaVisitor<ExecutionContext> javaVisitor = new AbstractRefasterJavaVisitor() {
-            final JavaTemplate before = Semantics.expression(this, "before", (String[] strings) -> String.join(", ", strings)).build();
-            final JavaTemplate after = Semantics.expression(this, "after", (String[] strings) -> String.join(":", strings)).build();
+            final JavaTemplate before = JavaTemplate
+                    .builder("String.join(\", \", foo.Arrays.before.strings)")
```

Comment:
This `foo.Arrays.before.strings` looks weird. Is that a parameter?

Reply:
Indeed the parameters look weird there for the test I added recently
https://github.com/openrewrite/rewrite-templating/blob/7993e10ca5ba483d4b56ac21d89a9811a3cfef9b/src/test/resources/refaster/Arrays.java#L21-L31